gammapy/maps/tests/test_hpxnd.py

```diff
@@ -505,4 +505,33 @@ def test_smooth(kernel):
         cutout.smooth(0.2 * u.deg, kernel)
 
     with pytest.raises(ValueError):
-        m_nest.smooth(0.2 * u.deg, "box")
\ No newline at end of file
+        m_nest.smooth(0.2 * u.deg, "box")
+
+def test_hpxmap_read_healpy(tmp_path):
+    path = tmp_path / "tmp.fits"
+
+    npix = 12 * 1024 * 1024
+    m = [np.arange(npix), np.arange(npix)-1, np.arange(npix)-2]
+    import healpy as hp
```

Comment:
Please move the import to the top of the method. Optional dependencies are imported "delayed" i.e. imported in the methods they are used like:
```python3

def test_something():
    import healpy as hp
    hp.write_map()
```